### DISCUSSION TITLE
Does Zellij have an equivalent to `DISABLE_AUTO_TITLE` in .zshrc files?

### DISCUSSION THREAD

**Discussion by ciaransweet** [zellij-org/zellij#2076] [Q&A]
As the title says, I like having my terminal *not* have titles.

See:

![image](https://user-images.githubusercontent.com/9111975/211607227-535e6fb0-fcf0-4f48-8da8-56fb66c9f64a.png)

However when I run `zellij` I get:

![image](https://user-images.githubusercontent.com/9111975/211607267-ff4ae03d-fc0f-4394-839b-33e8d40b3317.png)

Is there a similar property to `DISABLE_AUTO_TITLE` in `.zshrc` files for `zellij` where I can disable this behaviour?

Thanks!

**Comment by imsnif** [comment]
No such option, but if you'd like to open an issue it shouldn't be too hard to implement one.
